decoration.sk

```diff
@@ -255,7 +271,16 @@ decoratives after flattening:
 	(wet|moist|hydrated) farmland = minecraft:farmland[moisture=7]
 	{directional} {lightable} furnace¦s = minecraft:furnace
 	[any] furnace¦s = lit furnace, unlit furnace
-	snow layer¦s = minecraft:snow
+	
+	snow layer¦s = minecraft:snow_layer
+	(2|two) layer snow layer = minecraft:snow_layer[layers=2]
+	(3|three) layer snow layer = minecraft:snow_layer[layers=3]
+	(4|four) layer snow layer = minecraft:snow_layer[layers=4]
+	(5|five) layer snow layer = minecraft:snow_layer[layers=5]
+	(6|six) layer snow layer = minecraft:snow_layer[layers=6]
+	(7|seven) layer snow layer = minecraft:snow_layer[layers=7]
+	(8|eight) layer snow layer = minecraft:snow_layer[layers=8]
```

Comment:
Same as above, maybe use "thick" and add one for a specifically one layer thick one.